File: src/com/xj/bean/XjSign.java
package com.xj.bean;

import com.xj.annotation.NotInsertAnnotation;
import com.xj.annotation.NotUpdataAnnotation;

public class XjSign {
	
	@NotInsertAnnotation
	@NotUpdataAnnotation
	private Integer signId;
	
	private String nonce;
	
	private Integer createTime;

	public Integer getSignId() {
		return signId;
	}

	public void setSignId(Integer signId) {
		this.signId = signId;
	}

	public String getNonce() {
		return nonce;
	}

	public void setNonce(String nonce) {
		this.nonce = nonce;
	}

	public Integer getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Integer createTime) {
		this.createTime = createTime;
	}
}

File: WebRoot/js/xj-sign.js
var xjSign = function(map) {
	var str = "";
	var array = new Array();
	for(var key in map){
		array.push(key);
	}
	array.sort();
	var len = array.length;
	for(var i = 0 ; i < len ; i++){
		if(str == ""){
			var key = array[i];
			var value = map[value];
			str = key + "=" + value;
		}else{
			str = str + "," + key + "=" + value;
		}
	}
	return str;
};